test.js

```diff
@@ -1,11 +1,9 @@
+const tlds = require('.');
+const test = require('ava');
```

Comment:
There is no need for a testing framework here. This is hugely over-engineered. What's wrong with the previous `test.js` file?